core/ulog/README.md

```diff
@@ -0,0 +1,49 @@
+# Ulog package
+
+ulog provides an interface and abstraction layer over the logger implementation used underneath,
+and provides simple APIs for logging. The logger is instantiated as default logger and can be configured
+Configure() API and provided yaml configuration.
+
+```go
+package main
+
+import (
+  "go.uber.org/core/ulog"
```

Comment:
single imports would be a single line:

```go
import "go.uber.org/fx/core/ulog"
```

also, missing **fx** in the import path